docker/driver/Dockerfile

```diff
@@ -1,6 +1,16 @@
-FROM rust:1.31.0
+FROM fleupold/dfusion-solver-base:v1
 
+#Install Rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# Copy data
 WORKDIR /app
 COPY . .
+
+# Install solver if specified
```

Comment:
Maybe it would be wise to change the order here. Frist install rust dependencies and then add the solver. 

This might avoid rebuilding the rust dependences in  case of changes of the environmental variable $user_solver